src/test/java/org/folio/mr/api/MediatedRequestActionsApiTest.java

```diff
@@ -154,12 +158,30 @@ void mediatedRequestConfirmationForLocalInstanceAndRemoteItem() {
       .withHeader(HEADER_TENANT, equalTo(TENANT_ID_CENTRAL))
       .willReturn(jsonResponse(new ConsortiumItems().items(new ArrayList<>()), HttpStatus.SC_OK)));
 
+    User user = new User().id(UUID.randomUUID().toString());
+    wireMockServer.stubFor(WireMock.post(urlMatching(USERS_URL_TEMPLATE))
+      .withHeader(HEADER_TENANT, equalTo(TENANT_ID_CENTRAL))
+      .willReturn(jsonResponse(user, HttpStatus.SC_OK)));
+
+    wireMockServer.stubFor(WireMock.post(urlMatching(USERS_URL_TEMPLATE))
```

Comment:
why do we need a second stub for the same URL?

Reply:
Yes user is created twice - one for current tenant and one in central one.